### ISSUE TITLE
Crash when tapping on story circle

### ISSUE THREAD

**Issue by shmakovigor** [Netvent/storyly-mobile#52]
The app is crashing with the exception below when a user taps on the story circle inside StorylyView.

> Unable to activate constraint with anchors <NSLayoutYAxisAnchor:0x282dd2780 "UIStackView:0x12b80dc60.centerY"> and <NSLayoutYAxisAnchor:0x282dd3bc0 "Storyly.UIRoundImageView:0x12b816f20.centerY"> because they have no common ancestor.  Does the constraint or its anchors reference items in different view hierarchies?  That's illegal.

```
#0	0x00000001a6becc14 in objc_exception_throw ()
#1	0x00000001a6eaf158 in -[NSLayoutConstraint _setActive:mutuallyExclusiveConstraints:] ()
#2	0x00000001093bfb7c in ___lldb_unnamed_symbol872$$Storyly ()
#3	0x00000001093c0d34 in ___lldb_unnamed_symbol897$$Storyly ()
#4	0x00000001093c3174 in ___lldb_unnamed_symbol946$$Storyly ()
#5	0x00000001093c32a0 in ___lldb_unnamed_symbol947$$Storyly ()
#6	0x00000001093bea80 in ___lldb_unnamed_symbol849$$Storyly ()
#7	0x00000001093be7a8 in ___lldb_unnamed_symbol847$$Storyly ()
#8	0x00000001093ee244 in ___lldb_unnamed_symbol1906$$Storyly ()
#9	0x00000001093c85ec in ___lldb_unnamed_symbol1022$$Storyly ()
#10	0x00000001093c5a24 in ___lldb_unnamed_symbol992$$Storyly ()
#11	0x0000000193d38ab4 in -[UICollectionView _notifyWillDisplayCellIfNeeded:forIndexPath:] ()
#12	0x0000000193d392ec in -[UICollectionView _createPreparedCellForItemAtIndexPath:withLayoutAttributes:applyAttributes:isFocused:notify:] ()
#13	0x0000000193d3dc58 in -[UICollectionView _updateVisibleCellsNow:] ()
#14	0x0000000193d4264c in -[UICollectionView layoutSubviews] ()
#15	0x0000000194b7df84 in -[UIView(CALayerDelegate) layoutSublayersOfLayer:] ()
#16	0x00000001950987b4 in -[CALayer layoutSublayers] ()
#17	0x0000000195098c88 in CA::Layer::layout_if_needed(CA::Transaction*) ()
#18	0x00000001950ad47c in CA::Layer::layout_and_display_if_needed(CA::Transaction*) ()
#19	0x0000000194ff2a6c in CA::Context::commit_transaction(CA::Transaction*, double, double*) ()
#20	0x000000019501df34 in CA::Transaction::commit() ()
#21	0x0000000194663cec in _afterCACommitHandler ()
#22	0x0000000191bf8358 in __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ ()
#23	0x0000000191bf25c4 in __CFRunLoopDoObservers ()
#24	0x0000000191bf2b74 in __CFRunLoopRun ()
#25	0x0000000191bf221c in CFRunLoopRunSpecific ()
#26	0x00000001a96f6784 in GSEventRunModal ()
#27	0x0000000194630fe0 in -[UIApplication _run] ()
#28	0x0000000194636854 in UIApplicationMain ()
```

## Environment Details
**Storyly SDK Version:** 1.9.1
**iOS SDK Version:** 14.3
**Device Model:** iPhone 12 Pro Max
**Device OS Version:** 14.3

**Comment by levent-oral** [comment]
Hi @shmakovigor,

We're able to reproduce this issue by setting `storyHeaderIconIsVisible=false`. Can you confirm that this is the case? 

Thanks,

**Comment by shmakovigor** [comment]
Hi @levent-oral,

Yes, this is the settings I have.

```
storylyView.storylyInit = StorylyInit(storylyId: <###>, customParameter: <###>)
storylyView.rootViewController = self
storylyView.storyGroupSize = "small"
storylyView.delegate = self
storylyView.storyGroupListEdgePadding = 12
storylyView.storyHeaderIconIsVisible = false
```

**Comment by levent-oral** [comment]
Thank you @shmakovigor for confirmation! 

We'll release a patch version to fix this issue. We'll inform you.

**Comment by levent-oral** [comment]
Hello @shmakovigor, 

We've released an update which fixes this issue, [Storyly iOS SDK 1.9.2](https://github.com/Netvent/storyly-mobile/blob/master/ios/RELEASENOTES.md#192).

Please let us know if you have any further issues.

Thanks,
